packages/integrations/image/components/Picture.astro

```diff
@@ -26,10 +26,10 @@ const { src, sizes, widths, aspectRatio, formats = ['avif', 'webp'], loading = '
 const { image, sources } = await getPicture({ loader, src, widths, formats, aspectRatio });
 ---
 
-<picture {...attrs}>
+<picture>
 	{sources.map(attrs => (
 	<source {...attrs} {sizes}>))}
-	<img {...image} {loading} {decoding} />
+	<img {...image} {...attrs} {loading} {decoding} />
 </picture>
```

Comment:
How would this play out with Tailwind, which heavily uses the `class` attribute?

Reply:
`class` should be passed down to the `img` element here, so `<Picture class="rounded-sm" />` would result in `<picture><img class="rounded-sm" /></picture>`

I'd expect styling the `img` is the common case rather than styling the `picture` itself, but do you have a use case in mind?  Happy to reconsider exposing a way to add props to the `picture` and `img` separately if that's useful!

Reply:
Yea, this feels tough to understand for a user. If I add attribute `X`, how do I know which element that's being applied to? Could you apply it to both?

fwiw I know we discussed the other week that the `<Picture>` isn't as high of a priority as the other things on our plate for v1.0. I'd love to deprioritize anything `<Picture>` related and revisit once we're either closer to v1.0 or even post v1.0. 

Alternative, I could see a smaller quick fix that just moves the `alt` attribute to `img`, but leaves all remaining attributes as-is.

Reply:
Fair enough!  Rolled this back to only pass down the `alt` text to fix accessibility concerns, we can revisit the expected behavior for other attributes later

Reply:
@JuanM04 the current behaviour makes it impossible apply classes to the child `img` element, which was the main reason why I opened https://github.com/withastro/astro/issues/3909. The linked reproduction used Tailwind if you want to take a look.